src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Log/FunctionId.cs

```diff
@@ -502,5 +502,8 @@ internal enum FunctionId
         Workspace_Project_CompilationThrownAway = 460,
 
         CommandHandler_Paste_ImportsOnPaste = 470,
+
+        Find_Document_In_Workspace = 480,
```

Comment:
I should have mentioned this in my previous comment, but `_` are replaced with `/` in the telemetry event name, so for these you'd get something like vs/ide/vbcs/find/document/in/workspace

Would probably be more appropriate to just do FindDocumentInWorkspace / LSP_FindDocumentInWorkspace + LSP_RegisterWorkspace

Reply:
yeah I just took a look in telemetry explorer locally